config/routes.rb

```diff
@@ -388,28 +388,27 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      get '/by_mail' => 'by_mail/enter_code#index', as: :verify_by_mail_enter_code
-      post '/by_mail' => 'by_mail/enter_code#create'
+      get '/by_mail/enter_code' => 'by_mail/enter_code#index', as: :verify_by_mail_enter_code
+      post '/by_mail/enter_code' => 'by_mail/enter_code#create'
       get '/by_mail/confirm_start_over' => 'confirm_start_over#index',
           as: :confirm_start_over
 
       if FeatureManagement.gpo_verification_enabled?
-        get '/usps' => 'by_mail/request_letter#index', as: :request_letter
-        put '/usps' => 'by_mail/request_letter#create'
-
-        # These will be made the new "official" routes in a future commit
-        get '/by_mail/request_letter' => 'by_mail/request_letter#index'
+        get '/by_mail/request_letter' => 'by_mail/request_letter#index', as: :request_letter
         put '/by_mail/request_letter' => 'by_mail/request_letter#create'
+
+        # Temporary routes + redirects supporting GPO route renaming
```

Comment:
Is there a follow-up ticket to remove these temporary routes @matthinz ?

Reply:
👋 @matthinz Checking in again, I see these routes are still here. Did you create a follow-up ticket?